mycqu/enroll.py

```diff
@@ -0,0 +1,252 @@
+from __future__ import annotations
+
+import json
+import re
+from typing import List, Dict, Any, Optional, ClassVar, Tuple
+
+from ._lib_wrapper.dataclass import dataclass
+from .course import Course, CourseDayTime
+
+from requests import Session
+
+from .utils.datetimes import parse_weekday_str, parse_period_str, parse_weeks_str
+
+__all__ = ['EnrollCourseInfo', 'EnrollCourseTimetable', 'EnrollCourseItem']
```

Comment:
不需要改变值的情况下，元组占用内存更小一些（虽然这个更改在这里微不足道）
```suggestion
__all__ = ('EnrollCourseInfo', 'EnrollCourseTimetable', 'EnrollCourseItem')
```